src/components/molecules/ValidatePhoneForm.tsx

```diff
@@ -0,0 +1,26 @@
+import AppNextButton from "../atom/AppNextButton";
+import useValidatePhoneForm from "../../hooks/useValidatePhoneForm";
+import {useTranslation} from "react-i18next";
+import {namespaces} from "../../i18n/i18n.constants";
+import { AppDigitsValidator } from "../atom/AppDigitsValidator";
+
+const ValidatePhoneForm = () => {
+    const {validationCode, CODE_LENGTH, checkValidationCodeLength, onValidationCodeChange, onSubmit} = useValidatePhoneForm();
+    const {t: translate} = useTranslation(namespaces.pages.validatePhone);
+
+    return (
+        <form noValidate onSubmit={onSubmit} className="mt-4 flex w-full flex-col justify-between gap-4 self-center">
+            <div className="flex flex-col gap-4">
+                <AppDigitsValidator label={translate("Código de validación")} 
+                                    digitsCount={CODE_LENGTH} onChange={onValidationCodeChange} 
+                                    value={validationCode} />
+            </div>
+            {translate('resendCode') && (
+                <p className="mt-4 mb-2 text-right text-secondary-color text-sm underline cursor-pointer font-semibold">{translate('resendCode')}</p>
+            )}
```

Comment:
I don't understand this line, in which case do we get translate('resendCode') false? 

Why aren't we writing directly the paragraph instead of checking the resendCode text?
`<p className="mt-4 mb-2 text-right text-secondary-color text-sm underline cursor-pointer font-semibold">{translate('resendCode')}</p>`